web/src/features/public-api/types/observations.ts

```diff
@@ -89,6 +91,8 @@ export const transformDbToApiObservation = (
     observation;
 
   return {
+    usageDetails: {}, // Important: order matters here, in PG there are no usageDetails but in CH there are and will be written by rest
+    costDetails: {}, // Important: order matters here, in PG there are no costDetails but in CH there are and will be written by rest
```

Comment:
Did you check the API docs that this here + daily metrics are the only ones we need to check?